pkg/install/operator.go

```diff
@@ -406,13 +399,6 @@ func installLeaseBindings(ctx context.Context, c client.Client, namespace string
 	)
 }
 
-func installServiceBindings(ctx context.Context, c client.Client, namespace string, customizer ResourceCustomizer, collection *kubernetes.Collection, force bool) error {
-	return ResourcesOrCollect(ctx, c, namespace, collection, force, customizer,
-		"/rbac/operator-role-service-binding.yaml",
-		"/rbac/operator-role-binding-service-binding.yaml",
```

Comment:
It seems the `operator-role-binding-service-binding.yaml` file can also be removed.